Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift

```diff
@@ -89,6 +89,7 @@ class MessageQueueManagerImpl: MessageQueueManager {
     func getInlineMessages(forElementId elementId: String) -> [Message] {
         let messages = localMessageStore
             .filter { $0.value.elementId == elementId } // Only get messages for a specific elementid.
+            .filter { !hasMessageBeenShown($0.value) } // If we have already shown message, do not show again.
```

Comment:
Filter inline messages that have already been shown. 

This logic is identical to how modal messages are not shown if they have already been shown (inside of `showMessageIfMeetsCriteria`).